pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -38,6 +40,14 @@ func getDefaultInstrumentation(agentConfig *adapters.CwaConfig, isWindowsPod boo
 	if !ok {
 		return nil, errors.New("unable to determine dotnet instrumentation image")
 	}
+	autoInstrumentationConfigStr, ok := os.LookupEnv("AUTO_INSTRUMENTATION_CONFIG")
+	autoInstrumentationConfig := map[string]string{
+		"cpu":    "100m",
+		"memory": "64Mi",
```

Comment:
This is a placeholder default value, subject to change.

Reply:
As for defaults, I think it's more reasonable if we apply the same defaults from upstream. 

Ref
https://github.com/open-telemetry/opentelemetry-operator/blob/main/apis/v1alpha1/instrumentation_webhook.go#L101-L106

And can we make it different for each language?

Reply:
Oh cool, thanks for this. I'll discuss making it different with others.